src/routes/docs/products/messaging/topics/+page.markdoc

```diff
@@ -303,54 +292,34 @@ If you can't find the targets you'd like to add, see the [targets page](/docs/pr
 ```js
 const sdk = require('node-appwrite');
 
-// Init SDK
-const client = new sdk.Client();
+const client = new sdk.Client()
+    .setEndpoint('https://cloud.appwrite.io/v1')    // Your API Endpoint
+    .setProject('5df5acd0d48c2')                    // Your project ID
+    .setJWT('eyJhbVCJ9.eyJ...');                    // Your secret JSON Web Token
 
 const messaging = new sdk.Messaging(client);
 
-client
-    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
-    .setProject('5df5acd0d48c2')                           // Your project ID
-    .setJWT('eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ...') // Your secret JSON Web Token
-;
-
-const promise = messaging.createSubscriber(
-        '[TOPIC_ID]',                                      // topicId
-        '[SUBSCRIBER_ID]',                                 // subscriberId
-        '[TARGET_ID]'                                      // targetId
-    );
-
-promise.then(function (response) {
-    console.log(response);
-}, function (error) {
-    console.log(error);
-});
+const subscriber = await messaging.createSubscriber(
```

Comment:
Seems like there's some inconsistency with when we use `await` as it's not used in the examples above 🧐

Reply:
fixed